cluster-proportional-autoscaler.yaml

```
@@ -30,6 +30,17 @@ pipeline:
 
   - uses: strip
 
+subpackages:
+  - name: ${{package.name}}-compat
+    pipeline:
+      - runs: |
```

You can get rid of the `mkdir -p` by doing:

```suggestion
      - working-directory: ${{targets.subpkgdir}}
        runs: |
```